src/plugins/postal-savings-rs/index.ts

```diff
@@ -15,7 +15,9 @@ export const scrape: ScrapeFunc<Preferences> = async ({ preferences, fromDate, t
   for (const account of fetchedAccounts) {
     const rawAccountData = await fetchAccountData(account)
     accounts.push(convertAccount(account, rawAccountData))
-    transactions.push(...convertTransactions(account, rawAccountData))
+    const filteredTransactions = convertTransactions(account, rawAccountData)
+      .filter(transaction => transaction.date >= fromDate && transaction.date <= toDate)
```

Comment:
Should I write tests for it? If so, where should I put them?

Reply:
Basically there is no need to test such a simple code. But there seems to be an error. You haven't added transactions to the resulting transaction array :)

Reply:
Basically there is no need to write tests for such a simple code :)